packages/issuer-sdk-js/README.md

```diff
@@ -9,13 +9,123 @@ Get [our NPM package](https://www.npmjs.com/package/@idos-network/idos-sdk) and
+Passing ids are optional and will be auto-generated if not provided.
+
+```js
+import { createHuman } from "@idos-network/idos-issuer-sdk-js";
+import issuerConfig from "./issuer-config.js";
+
+const human = {
+  id: crypto.randomUUID(),
+  current_public_key: "DERIVED_PUBLIC_KEY",
+}
+
+const walletPayload = {
+  id: crypto.randomUUID(),
+  address: "0x0",
+  wallet_type: "EVM",
+  message: "SIGN_MESSAGE",
+  signature: "DERIVED_SIGNATURE",
+  public_key: "DERIVED_PUBLIC_KEY",   
+}
+
+// Will return a tuple with the human and the wallet associated to the human.
+const [profile, wallet] = await createHuman(issuerConfig, human, walletPayload); 
+```
+
+## Writing credentials.
+
+In order to write a credential to idOS, the issuer needs to obtain permission from the user. This can be done in two ways: either through a permissioned approach or by using a write grant. Below are the two methods for writing credentials.
+
+#### 1. Using Write Grant
+The first method involves getting permission from the user via a Write Grant. This grants the issuer the ability to write to the user’s profile. To do this, you must first create a Write Grant using the idOS SDK.
+
+Example:
+<div >
+ you can create a write grant by calling the  <a href="https://github.com/idos-network/idos-sdk-js/tree/main/packages/idos-sdk-js#write-grants">addWriteGrant</a>
+</div>
+
+```js
+import { idOS } from "@idos-network/idos-sdk-js";
+
+const sdk = await idOS.init(...);
+
+const granteeAddress = "0x0"; // The address of the grantee.
+
+// Create a write grant.
+await sdk.data.addWriteGrant(granteeAddress);
+```
+This will create a write grant in the idOS for the given grantee address. Now, the issuer can create a credential:
+
+```js
+import { createCredentialByGrant } from "@idos-network/idos-issuer-sdk-js";
+import issuerConfig from "./issuer-config.js";
+
+const credential = {
+  id: crypto.randomUUID(),
+  content: "CREDENTIAL_CONTENT", // The credential content should be passed as is. It will be encrypted for the recipient before being stored on the idOS.
+  encryption_public_key: "ENCRYPTION_PUBLIC_KEY", // The public key of the recipient.
+  // Other credential params
+  ...
+}
+
+const credential = await createCredentialByGrant(issuerConfig, credential);
+```
+
+This will create a credential in the idOS for the given grantee address.
+
+<div style="background-color: #ffffcc; border-left: 6px solid #ffeb3b; padding: 10px;">
+  <strong>Important:</strong>The credential content should be passed as is. It will be encrypted for the recipient before being stored on the idOS.
+</div>
+
+#### 2. Using Permissioned Credential Creation
+The second method allows the issuer to create a credential without a Write Grant by having a permissioned approach. This method assumes the issuer already has direct permission to write the credential.
+
+In this case, the ``createtCredentialPermissioned`` function is used to write the credential with necessary encryption.
```

Comment:
Huh. I was not aware that double-backticks would work!

Was there any reason in particular to use them? If not, I'd rather single backticks to keep things consistent and lessen surprises and double-guessing for future editors.

Reply:
my bad. no special reason. just fixed them